steps/git-and-gh/install.sh

```diff
@@ -4,41 +4,46 @@ print_git_and_gh_documentation() {
   _print_step_documentation "${REPO_DIR}/steps/git-and-gh"
 }
 
-set_git_username() {
-  read -r username
-  git config --global user.name "${username}"
-}
-
-get_git_username() {
+get_git_author_name() {
   git config --get user.name
 }
 
-set_git_user_email() {
-  echo 'git config --global user.email "your_email@freeagent.com"'
-  read -r email
-  git config --global user.email "${email}"
+set_git_author_name() {
+  echo 'Configuring the name you'"'"'d like to associate with your commits' >&2
```

Comment:
Gotta love escaping quotes 😅
```suggestion
  echo "Configuring the name you'd like to associate with your commits" >&2
```